Source/Core/InputCommon/ControllerEmu/ControlGroup/Cursor.h

```diff
@@ -25,9 +25,10 @@ class Cursor : public ReshapableInput
 
   Cursor(std::string name, std::string ui_name);
 
-  ReshapeData GetReshapableState(bool adjusted) final override;
+  ReshapeData GetReshapableState(bool adjusted) const final override;
   ControlState GetGateRadiusAtAngle(double ang) const override;
 
+  // Modifies the state
   StateData GetState(bool adjusted);
```

Comment:
This seems like a weird choice, having _some_ `GetState` members as `const` and others as _not_ `const`. By the name, they probably _should_ be, but I guess this would be out of scope here since it seems a bit more involved for the Cursor input.

Reply:
See my answer to the other similar comment you made. Still don't understand when you say: out of scope.
Worth nothing that I tried to quickly make these const, but it was too complicated, it can be done in the future.

Reply:
With "out of scope" i meant "rewriting `GetState` to be `const`-able. Your other PR does something with a `bool update` parameter that makes it clear; and it is probably better suited for something like that (if you _do_ want to make this more explicit than just a comment)

Reply:
For this specific GetState() on the Cursor, I have also made changes to it, which were already in this PR:
https://github.com/dolphin-emu/dolphin/blob/d41f881b5b1e1579fb4a3bc40d769696bee71e00/Source/Core/InputCommon/ControllerEmu/ControlGroup/Cursor.h#L37
but will come in a more polished version in one of my upcoming PRs.

So it will kind of also have a `bool update`